Update `defaultAvatar` calculation

zaza7 opened the pull request:
You thought I wouldn't but I would, Abal. :^)

zaza7 committed:
Fix default avatar function to return what jake said it should return :^)

zaza7 committed:
Remove DefaultAvatarHashes

forgot to add this file to the initial commit, we smart, don't worry.

abalabahaha commented on lib/structures/User.js:
```diff
@@ -41,11 +41,11 @@ class User extends Base {
 
     get defaultAvatarURL() {
-        return `https://discordapp.com/assets/${this.defaultAvatar}.png`;
+        return `https://cdn.discordapp.com/embed/avatars/${this.defaultAvatar}.png`;
```
Use `CDN_URL`, see below for examples

zaza7 committed:
Use CDN_URL

Even though this was supposed to be a meme, I guess it's actually a helpful PR after all of it.
